analyzers/tests/SonarAnalyzer.UnitTest/TestCases/PreferGuidEmpty.vb

```diff
@@ -0,0 +1,87 @@
+﻿Imports System
+
+Class Compliant
```

Comment:
In general, same comments apply here as the ones in the C# variant. Except the lower-case fields :)